discord/message.py

```diff
@@ -772,7 +779,9 @@ def system_content(self):
         except KeyError:
             pass
         else:
-            await self._state.http.suppress_message_embeds(self.channel.id, self.id, suppress=suppress)
+            if suppress:
+                self.flags._set_flag(4, True)
```

Comment:
This entire implementation is weird. For starters, this is a boolean so you can do `suppress=True` and `suppress=False`. This code only handles if it's `suppress=True`. Likewise, we should probably support passing in a `MessageFlags` value directly.